PopPUNK/models.py

```diff
@@ -1080,19 +1120,29 @@ def fit(self, X, accessory):
         else:
             self.dist_col = 0
 
-        self.nn_dists = {}
+        print("Sparsify distances")
```

Comment:
We normally write to stderr

Reply:
Leftover debug print, thanks for spotting, gone in 2a53e80